core/src/main/java/org/frankframework/senders/JavascriptSender.java

```diff
@@ -39,10 +41,8 @@
 import org.frankframework.util.Misc;
 import org.frankframework.util.StreamUtil;
 
-import lombok.Getter;
-
 /**
- * Sender used to run JavaScript code using J2V8
+ * Sender used to run JavaScript code using J2V8 or GraalJS.
```

Comment:
```suggestion
 * Sender used to run JavaScript code using `JavascriptEngine` implementations.
```

Mocht er dan ooit nog een implementatie komen, hoef je dit niet meer aan te passen ;-)